packages/fluentui/react/src/components/Input/Input.tsx

```diff
@@ -185,13 +190,23 @@ class Input extends AutoControlledComponent<WithAsProp<InputProps>, InputState>
 
   handleIconOverrides = predefinedProps => ({
     onClick: (e: React.SyntheticEvent) => {
+      if (this.props.disabled) {
+        e.preventDefault();
+        return;
+      }
+
       this.handleOnClear(e);
       this.inputRef.current.focus();
       _.invoke(predefinedProps, 'onClick', e, this.props);
     }
   });
 
   handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (this.props.disabled) {
+      e.preventDefault();
+      return;
+    }
+
```

Comment:
```suggestion
```



Can you please explain why we need this? It's not possible to dispatch a `change` for `disabled` `input` even manually:
https://codesandbox.io/s/blue-architecture-2l2i4